src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -32,6 +32,16 @@
     <LangVersion Condition="'$(LangVersion)' == '' AND '$(_MaxSupportedLangVersion)' != ''">$(_MaxSupportedLangVersion)</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
Should this be in Microsoft.Managed.Core.targets so that it's shared by both C# and VB? (for the extension you can use `$(DefaultLanguageSourceExtension)`)